std/portable/index.js

```diff
@@ -259,21 +259,25 @@ globalScope["fmodf"] = function fmodf(x, y) {
 
 globalScope["JSMath"] = Math;
 
-Object.defineProperties(globalScope["JSMath"], {
-  sincos_sin: { value: 0.0, writable: true },
-  sincos_cos: { value: 0.0, writable: true },
-  signbit: {
-    value: function signbit(x) {
-      F64[0] = x; return Boolean((U64[1] >>> 31) & (x == x));
-    }
-  },
-  sincos: {
-    value: function sincos(x) {
-      this.sincos_sin = Math.sin(x);
-      this.sincos_cos = Math.cos(x);
+try {
```

Comment:
Why you wrap this to try/catch?

Reply:
Ah because since my code depends on portable too, it causes it to be redefined.  I suppose I could change this to check if already define.